docs/slack-connector/installation.md

```diff
@@ -1,13 +1,64 @@
 # Slack Connector Installation<!-- omit in toc -->
 
 - [Overview](#overview)
-- [Installation in Kyma with Helm](#installation-in-kyma-with-helm)
+- [Installation in Kyma as Add-On](#installation-in-kyma-as-add-on)
   - [Prerequisites](#prerequisites)
   - [Steps](#steps)
+  - [Verification](#verification)
+  - [Removal](#removal)
+- [Installation in Kyma with Helm](#installation-in-kyma-with-helm)
+  - [Prerequisites](#prerequisites-1)
+  - [Steps](#steps-1)
 
 ## Overview
 
-The Slack Connector is a component which allows contact from inside of Kyma environment to the Slack API.
+The Slack Connector is a component which allows interaction with Slack API from inside of Kyma environment. The simplest way to install Slack Connector in Kyma is to install it as Add-On.
+
+## Installation in Kyma as Add-On
+
+### Prerequisites
+
+- Slack Bot with desired privileges installed to destination workspace. Tutorial provided by Slack on how to setup an application can be found [here](https://api.slack.com/bot-users#getting-started).
+  - Slack Bot Token (only if you want to send messages to Slack Web API).
+  - Slack Signed Secret (only if you want to receive events to Slack Event API).
```

Comment:
In Slack settings it is called Signing Secret, not Signed. It could be misleading
```suggestion
  - Slack Signing Secret (only if you want to receive events to Slack Event API).
```

Reply:
Done